src/Workspaces/Core/Portable/CodeFixesAndRefactorings/EnumArrayConverter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.CodeFixesAndRefactorings;
+
+internal class EnumArrayConverter
+{
+    public static TEnum[] FromStringArray<TEnum>(string[] strings) where TEnum : struct
```

Comment:
```suggestion
    public static TEnum[] FromStringArray<TEnum>(string[] strings) where TEnum : struct, System.Enum
```

Reply:
Nite: consider returning ImmutableArray instread if you don't need to mutate hte array.